tests/bug-1449-01/test.yaml

```diff
@@ -0,0 +1,17 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        alert.signature_id: 2220017
+  - filter:
+      count: 1
+      match:
+        event_type: smtp
+        smtp.helo: bug.client
+        email.status: PARSE_ERROR
```

Comment:
Why do we want to enforce failure when we can succeed with the rust version ? (without overflowing)

Reply:
point here is to describe existing behavior. We won't update 6 or 7 parsers to rust

Reply:
So, should the check be done with lt-version: 8 ?

Reply:
I guess only when behavior for 8 changes